packages/app-aco/src/sorting.tsx

```diff
@@ -0,0 +1,24 @@
+import orderBy from "lodash/orderBy";
+
+import { ListDbSort, ListTableSort, ListTableSortDirection } from "~/types";
+
+export const handleDbSorting = (initial?: ListDbSort): ListDbSort => {
```

Comment:
5 files into the PR, I finally got to the function body. This function validates the sorter, and returns a default sort if parameter is invalid. Name it that: `getValidSort`, or `validateOrGetDefaultSort`, just some ideas.

Reply:
👍 I changed it into `validateOrGetDefaultDbSort`, it explains better what the method does.